packages/core/src/App/Containers/Layout/header/traders-hub-onboarding.tsx

```diff
@@ -4,12 +4,24 @@ import { Icon, Popover } from '@deriv/components';
 import { routes } from '@deriv/shared';
 import { observer, useStore } from '@deriv/stores';
 import { Localize } from '@deriv/translations';
+import { useFeatureFlags } from '@deriv/hooks';
 
 const TradersHubOnboarding = observer(() => {
     const history = useHistory();
     const { traders_hub, ui } = useStore();
     const { setIsOnboardingVisited } = traders_hub;
     const { is_dark_mode_on } = ui;
+    const { is_next_wallet_enabled } = useFeatureFlags();
+
+    const onClickHandler = is_next_wallet_enabled
+        ? () => {
+              localStorage.setItem('walletsOnboarding', 'started');
+              window.dispatchEvent(new Event('storage'));
+          }
+        : () => {
+              history.push(routes.onboarding);
+              setIsOnboardingVisited(false);
+          };
```

Comment:
```suggestion
    const onClickHandler = is_next_wallet_enabled
        ? () => {
              localStorage.setItem('walletsOnboarding', 'started');
              window.dispatchEvent(new Event('storage'));
          }
        : () => {
              history.push(routes.onboarding);
              setIsOnboardingVisited(false);
          };
```

Please use `useLocalStorage` hook from our installed `usehooks-ts`: 

https://usehooks-ts.com/react-hook/use-local-storage

This already have the StorageEvent handled so all instances where you use `useLocalStorage` hook will be notified on storage change 👍🏼